src/test/BundleTests/Microsoft.NET.HostModel.Bundle.Tests/BundlerConsistencyTests.cs

```diff
@@ -143,6 +143,26 @@ public void AllBundledFilesAreExtracted()
             bundleDir.Should().OnlyHaveFiles(expectedFiles);
         }
 
+        [Fact]
+        public void TestAssemblyAlignment()
+        {
+            var fixture = sharedTestState.TestFixture.Copy();
+
+            var hostName = BundleHelper.GetHostName(fixture);
+            var appName = Path.GetFileNameWithoutExtension(hostName);
+            string publishPath = BundleHelper.GetPublishPath(fixture);
+            var bundleDir = BundleHelper.GetBundleDir(fixture);
+
+            var bundler = new Bundler(hostName, bundleDir.FullName);
+            bundler.GenerateBundle(BundleHelper.GetPublishPath(fixture));
+
+            foreach(var fileEntry in bundler.BundleManifest.Files)
+            {
+                Assert.True(!Bundler.NeedsAlignment(fileEntry.Type) || 
```

Comment:
I think the test should not rely on the product to tell it what needs alignment... since we know exactly what app we're building here, the test should relatively easily list all the files to be aligned.